index.html

```diff
@@ -99,64 +130,96 @@ <h1 class="card-title card-title-1">Multi-Post Stories</h1>
-                        </ul>
-                    </div>                    
-                    <button type="button" class="project-detail-button clickable">See Project</button>
+                    <div class="project-card">
+                        <div class="card-body background-1">
+                            <h1 class="card-title">Professional Art Printing Data</h1>
+                            <p class="card-content">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul class="tags-list">
+                                <li class="tag">HTML</li>
+                                <li class="tag">CSS</li>
+                                <li class="tag">JavaScript</li>
+                            </ul>
+                        </div>                    
+                        <button type="button" class="project-detail-button clickable">See Project</button>
+                    </div>
+    
+                    <div class="project-card">
+                        <div class="card-body background-2">
+                            <h1 class="card-title">Professional Art Printing Data</h1>
+                            <p class="card-content">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul class="tags-list">
+                                <li class="tag">HTML</li>
+                                <li class="tag">CSS</li>
+                                <li class="tag">JavaScript</li>
+                            </ul>
+                        </div>                    
+                        <button type="button" class="project-detail-button clickable">See Project</button>
+                    </div>
+    
+                    <div class="project-card">
+                        <div class="card-body background-3">
+                            <h1 class="card-title">Professional Art Printing Data</h1>
+                            <p class="card-content">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul class="tags-list">
+                                <li class="tag">HTML</li>
+                                <li class="tag">CSS</li>
+                                <li class="tag">JavaScript</li>
+                            </ul>
+                        </div>                    
+                        <button type="button" class="project-detail-button clickable">See Project</button>
+                    </div>
                 </div>
             </div>
             <!-- projects list end -->
```

Comment:
- [ ] Great job :call_me_hand: For your About Myself section I noticed a few differences between your design and Figma :cry:  . Please make sure that you stick to the design as much as possible (e.g., font, colors, images, text, margins) because it is very important to pay attention to details as a developer +1
Check the screenshots below to see the differences :arrow_down: 

#### Figma design
![Screenshot from 2022-09-29 21-08-23](https://user-images.githubusercontent.com/6615394/193143638-0af42a1e-ba13-4c1b-bd0e-862d3fb8359b.jpg)

#### Your design
![Screenshot from 2022-09-29 21-08-46](https://user-images.githubusercontent.com/6615394/193143650-c6dc48c9-3a14-41ce-b386-0b79c8d4b785.png)